### DISCUSSION TITLE
Running code lens commands

### DISCUSSION THREAD

**Discussion by wintersteiger** [emacs-lsp/lsp-mode#4416] [Q&A]
What are the keyboard shortcuts or command names to select and run a code lens command?

**Comment by jcs090218** [comment]
I don't think it's bind into any key yet. You can manually trigger it with `M-x lsp-lens-show`.

**Comment by yyoncho** [comment]
Install avy and then `M-x lsp-avy-lens`

**Comment by wintersteiger** [comment]
Neither of those suggestions work for me. It may be due to the fact that I'm running a text-mode emacs. `M-x lsp-lens-show` doesn't do anything at all and `M-x lsp-avy-lens` starts, but avy won't let me select any commands.

**Reply by yyoncho** [reply]
Maybe you need M-x lsp-execute-code-action?
